src/libs/Navigation/NavigationRoot.tsx

```diff
@@ -50,11 +49,11 @@ function parseAndLogRoute(state) {
     Navigation.setIsNavigationReady();
 }
 
-function NavigationRoot(props) {
+function NavigationRoot({authenticated, onReady}: NavigationRootProps) {
     useFlipper(navigationRef);
     const firstRenderRef = useRef(true);
 
-    const {updateCurrentReportID} = useCurrentReportID();
+    const currentReport = useCurrentReportID();
```

Comment:
Can we have better name something like `currentReportIDValue`? Obviously, this is not report.

Reply:
I agree 👍 Provided name makes more sense